clients/include/flops.hpp

```diff
@@ -23,16 +23,68 @@ constexpr double axpy_gflop_count(rocblas_int n)
 {
     return (2.0 * n) / 1e9;
 }
-template <typename T>
+template <>
+constexpr double axpy_gflop_count<rocblas_float_complex>(rocblas_int n)
+{
+    return (8.0 * n) / 1e9; // 6 for complex-complex multiply, 2 for c-c add
+}
+template <>
+constexpr double axpy_gflop_count<rocblas_double_complex>(rocblas_int n)
+{
+    return (8.0 * n) / 1e9;
+}
+
+template <bool CONJ, typename T>
 constexpr double dot_gflop_count(rocblas_int n)
 {
     return (2.0 * n) / 1e9;
```

Comment:
Technically this should be `n + (n - 1)` since there only has to be `n - 1` adds, right? Although I'm guessing this is nitpicking and we don't care haha.

Reply:
Agreed... better to be more accurate though I suppose?

Reply:
Most algorithms initialize `sum =  0` and add to it `n` times, which can handle the case when `n==0`. So it's `n` adds, not `n-1`.